source/Calamari.Tests/Fixtures/PackageRetention/JsonJournalRepositoryFixture.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.IO;
+using Calamari.Common.Features.Processes.Semaphores;
+using Calamari.Common.Plumbing.Deployment.PackageRetention;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Deployment.PackageRetention.Model;
+using Calamari.Deployment.PackageRetention.Repositories;
+using Calamari.Tests.Fixtures.Integration.FileSystem;
+using Calamari.Tests.Helpers;
+using FluentAssertions;
+using NSubstitute;
+using NUnit.Framework;
+
+namespace Calamari.Tests.Fixtures.PackageRetention
+{
+    [TestFixture]
+    public class JsonJournalRepositoryFixture
+    {
+        readonly string testDir = TestEnvironment.GetTestPath("Fixtures", "JsonJournalRepository");
+
+        [SetUp]
+        public void SetUp()
+        {
+            if (!Directory.Exists(testDir))
+                Directory.CreateDirectory(testDir);
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            if (Directory.Exists(testDir))
+                Directory.Delete(testDir, true);
+        }
+
+        [TestCase("PackageRetentionJournal.json")]
+        [TestCase(null)]
```

Comment:
I think it would be good to use the TestName or Description parameters of the TestCase attribute here, just so we know what we're actually testing here with these values.

Reply:
Oh I completely missed that from the page you linked 🤦 yeah that would be nice haha

Reply:
Added a TestName for both